hs_core/views/resource_folder_hierarchy.py

```diff
@@ -0,0 +1,313 @@
+import json
+import logging
+import os
+
+from django.http import HttpResponse
+from rest_framework.exceptions import NotFound, PermissionDenied

```

Comment:
@pkdash In fact, django.http is the only django import here, and the next import is django_irods module, so no change of import order is needed here.